orchagent/orch.h

```diff
@@ -18,6 +18,7 @@ using namespace std;
 using namespace swss;
 
 const char delimiter           = ':';
+const char delimiter2          = '|';
```

Comment:
delimiter2 doesn't tell what it is separating, can you give it a more meaningful name? like field_delimiter if it is separating fields, or attr_delimiter if it is separating attributes, for example.